docs/en/essentials/named-views.md

```diff
@@ -27,3 +27,58 @@ const router = new VueRouter({
 ```
 
 A working demo of this example can be found [here](https://jsfiddle.net/posva/6du90epg/).
+
+## Nested Named Views
+
+While being quite complex, it is possible to combine named views with nested view. When doing so, you will also need to name nested `router-view` components used. Let's take a Settings panel example:
```

Comment:
This sentence should be re-worded.
---
It is possible to create complex layouts using named views with nested views. For example, let us see following example; a settings panel.